Os/Posix/IPCQueue.cpp

```diff
@@ -68,7 +68,7 @@ namespace Os {
         handle = mq_open(this->m_name.toChar(), O_RDWR | O_CREAT | O_EXCL, 0666, &att);
 
         // If queue already exists, then unlink it and try again.
-        if (-1 == (NATIVE_INT_TYPE) handle) {
+        if (-1 ==  handle) {
```

Comment:
Does this compare work cleanly between mqd_t and int?

Reply:
mq_t is a signed value that can return -1 on error.